client/src/variable.scss

```diff
@@ -0,0 +1,59 @@
+// Variable für Font-Family's:
+$ff-roboto: "Roboto", Arial, sans-serif;
+
+// Variablen für Global-Colors:
+$gc-1: #40AA54;
+$gc-2: #FFFFFF;
+$gc-3: #000000;
+$gc-4: #777777;
+
+// BackgroundColor:
+$bgc: #F5F5F5;
+
+// Variablen für Font-Colors:
+// Farben ändern, falls Dark-Mode eingebaut wird
+$fc-1: #40AA54;
+$fc-2: #FFFFFF;
+$fc-3: #000000;
+$fc-4: #777777;
+$fc-5: #2C3066;
```

Comment:
mit so zahlen ist es jetzt mal okay. In einem echten Szenario aber super schwer damit zu arbeiten, weil man nicht auf anhieb weiß wofür die Zahlen stehen.

Reply:
meinst du 1,2,3,4,5?? da hast du recht. das hatte ich mir auch gedacht, wir hatten aber zu beginn doch mal gesagt, dass wir den farbstil nochmal neu anpassen, evtl auch komplett dunkel. deshalb habe ich keine namen sondern zahlen genutzt, hat für mich mehr sinn ergeben, falls wir die farben ersetzen sollten